internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -64,35 +66,32 @@ type FBCRegistryPod struct { //nolint:maligned
 	// FBCContent represents the contents of the FBC file (string YAML).
 	FBCContent string
 
-	// FBCDir is the name of the FBC directory name where the FBC resides in.
-	FBCDir string
-
-	// FBCFile represents the FBC filename that has all the contents to be served through the registry pod.
-	FBCFile string
+	// FBCIndexRootDir is the FBC directory that exists under root of an FBC container image.
+	// This directory has the File-Based Catalog representation of a catalog index.
+	FBCIndexRootDir string
 
 	cfg *operator.Configuration
 }
 
 // init initializes the FBCRegistryPod struct.
-func (f *FBCRegistryPod) init(cfg *operator.Configuration) error {
+func (f *FBCRegistryPod) init(cfg *operator.Configuration, cs *v1alpha1.CatalogSource) error {
 	if f.GRPCPort == 0 {
 		f.GRPCPort = defaultGRPCPort
 	}
 
+	if f.FBCIndexRootDir == "" {
+		f.FBCIndexRootDir = "/configs"
+	}
```

Comment:
This if would _always_ evaluate to true. IMO we can remove the variable altogether and use the constant in the container creation.